packages/workbox-build/src/templates/sw-template.js

```diff
@@ -14,23 +14,41 @@
   limitations under the License.
 */
 
-module.exports = `<% if (importScripts) { %>
+module.exports = `/**
+ * Welcome to your Workbox-powered service worker!
+ *
+ * Here are some next steps:
+ *
+ * - Your web app needs to register this file.
+ *   See https://goo.gl/DNGzMp
+ *
+ * - Disable HTTP caching for this file.
+ *   See https://goo.gl/rWuKgq
+ *
+ * The rest of the code in this file is auto-generated. Please don't update this
```

Comment:
Nit: drop "in this file" on this line, reading from top to bottom there is "this file" referenced in previous bullet points and sounds repitive here.